src/library_pipefs.js

```diff
@@ -0,0 +1,208 @@
+mergeInto(LibraryManager.library, {
+  $PIPEFS__postset: '__ATINIT__.push(function() { PIPEFS.root = FS.mount(PIPEFS, {}, null); });',
```

Comment:
That is a good observation. 

Reading closer, it looks like `FS.init.initialized` is not needed to be true in order to be able to `FS.mount()` (`FS` does that itself in `FS.staticInit()` as well) However `FS.staticInit()` must have been called before `FS.mount()` can. I think if this does not hold, then errors will naturally manifest by the absence of `FS.nameTable`, so things should be good here.